test/support/idx/factories/authenticators.ts

```diff
@@ -134,7 +134,14 @@ export const SecurityQuestionAuthenticatorFactory = IdxAuthenticatorFactory.para
   type: 'security_question',
   methods: [
     { type: 'security_question' }
-  ]
+  ],
+  // if already enrolled, contextualData will be set
```

Comment:
remove this comment?

Reply:
I put this in there deliberately, because it seems like the kind of thing we will forget. By default the factory will build an object for enrollment, but if you want to test verification you'll need to add that in yourself.